taca_ngi_pipeline/cli.py

```diff
@@ -61,31 +76,36 @@ def project(ctx, projectid):
 def sample(ctx, projectid, sampleid):
     """ Deliver the specified sample to the specified destination
     """
+    if ctx.parent.params['cluster'] == 'bianca':
+        #in this case I need to open a sftp connection in order to avoid to insert password everytime
+        projectObj = _deliver_castor.CastorProjectDeliverer(projectid, **ctx.parent.params)
+        projectObj.create_sftp_connnection()
+        #create the project folder in the remote server
+        #move to delivery folder
+        projectObj.sftp_client.chdir('wharf')
```

Comment:
In the module, you're referring to `castordeliverypath` but not here. Can it be harmonized?